nmostesting/mocks/Registry.py

```diff
@@ -622,6 +626,41 @@ def post_subscription(version):
     return jsonify(subscription_response), status_code, {"Location": location}
 
 
+@REGISTRY_API.route('/x-nmos/query/<version>/subscriptions/<subscription_id>', methods=["DELETE"])
+def delete_subscription(version, subscription_id):
+
+    registry = REGISTRIES[flask.current_app.config["REGISTRY_INSTANCE"]]
+    if not registry.enabled:
+        abort(503)
+    authorized = registry.check_authorized(request.headers, request.path)
+    if authorized is not True:
+        abort(authorized)
+
+    registry.requested_query_api_version = version
+
+    try:
+        subscription = next(iter([subscription for id, subscription in registry.get_resources()['subscription'].items()
```

Comment:
Please would you explain what `next(iter(...), None)` is doing?

Reply:
File it under "cut and paste errors" I think. Now changed to something simpler.